proto/buf.gen.yaml

```diff
@@ -2,13 +2,14 @@ version: v1
 managed:
   enabled: true
   go_package_prefix:
-    default: go.buf.build/grpc/go/conduitio/conduit-connector-protocol
+    default: "github.com/conduitio/conduit-connector-protocol/proto/gen"
 plugins:
-  - name: go
-    out: ../internal
+  - plugin: buf.build/protocolbuffers/go:v1.28.1
+    out: gen
```

Comment:
If you want we can change the `out` property to `.` and remove `/gen` from the go package prefix. Related to our [discussion here](https://github.com/ConduitIO/conduit/pull/927#discussion_r1129574773).